### PR TITLE
09_dt_rts: execution results

### PR DESCRIPTION
**Pull request by ASynenko**
Patches for rts and eeprom. Logs from board

### CODE REVIEW

File: 09_dt_rtc/eeprom_ops.log [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+00006000  73 6f 6d 65 20 74 6f 74  61 6c 79 20 72 61 6e 64  |some totaly rand|
+00006010  6f 6d 65 20 74 65 78 74  0a ff ff ff ff ff ff ff  |ome text........|
+00006020  ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff  |................|
+*
+00007000  73 6f 6d 65 20 74 6f 74  61 6c 79 20 72 61 6e 64  |some totaly rand|
+00007010  6f 6d 65 20 74 65 78 74  0a ff ff ff ff ff ff ff  |ome text........|
+00007020  ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff  |................|
+*
+00008000
+/ #
```
Comment (ReAlUA) [inline 09_dt_rtc/eeprom_ops.log:56]: From task description:
---
* Rebuild  .dtb file and copy it to network boot directories.
* Use `cat /the/directory/you/found/compatible` to check if type is actually changed.
* Read the whole EEPROM device file again and compare results.

File: 09_dt_rtc/i2c_detection.log [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+60: -- -- -- -- -- -- -- -- 68 -- -- -- -- -- -- -- 
+70: -- -- -- -- -- -- -- --                         
+/ # 
+/ # i2cdump -y -r 00h-06h 2 68 
+i2cdump: invalid range
+/ # i2cdump -y -r 00-06 2 68 
+     0  1  2  3  4  5  6  7  8  9  a  b  c  d  e  f    0123456789abcdef
+00: 42 37 14 05 18 03 21                               B7????!         
+/ # 
+/ #
```
Comment (ReAlUA) [inline 09_dt_rtc/i2c_detection.log:46]: From the task description:
---
* Using i2ctools (i2cdetect, i2cdump, etc.) read *and write* date/time registers 00h-06h, *start and stop clock* using EOSC bit in register 0Eh.

File: 09_dt_rtc/rtc_ops.log [hunk 3]
```diff
@@ -0,0 +1,33 @@
+/ # 
+/ # 
+/ # dmesg | grep 'rtc'
+[    2.079035] rtc-ds1307 2-0068: registered as rtc0
+[    2.161474] rtc-ds1307 2-0068: setting system clock to 2021-03-18 14:49:21 UTC (1616078961)
+/ # 
+/ # 
+/ # date -s "2021-03-18 14:50:00" 
+Thu Mar 18 14:50:00 UTC 2021
+/ # hwclock -w -f /dev/rtc0
```
Comment (ReAlUA) [inline 09_dt_rtc/rtc_ops.log:10]: Just note: `date -s` should set time in the default hw clock. You added rtc in the device tree and the module is loaded during boot time.

**Review by ReAlUA — CHANGES_REQUESTED** [review]

File: 09_dt_rtc/eeprom_ops.log (hunk 1 above)
Comment (ASynenko) [reply 09_dt_rtc/eeprom_ops.log:56]: Done

File: 09_dt_rtc/i2c_detection.log (hunk 2 above)
Comment (ASynenko) [reply 09_dt_rtc/i2c_detection.log:46]: Done.

**Review by ReAlUA — APPROVED** [review]

**Pull request merged**